packages/puppeteer/package.json

```diff
@@ -44,7 +44,7 @@
     "express": "^4.18.2",
     "mocha": "^10.0.0",
     "nyc": "^15.1.0",
-    "puppeteer": "^19.7.3",
+    "puppeteer": "19.7.3",
```

Comment:
Starting in Puppeteer 19.11 (ish?) they started adding a log that stated that `headlness: new` would be needed (which massively clogs up the test run as it logs a huge paragraph every test). However, if you add `headless: new` to the options it breaks the unloaded iframe test. It seems that whatever is in `headless: new` fixes the problem with unloaded iframes? Will open a tech debt ticket to investigate.

Reply:
https://github.com/dequelabs/axe-core-npm/issues/828